fix: update MQTT Examples for Ireland

jenmwms opened the pull request:
Closes edgexfoundry#546

Signed-off-by: Elizabeth J Lee <elizabeth.j.lee@intel.com>
Signed-off-by: Jennifer Williams <jennifer.m.williams@intel.com>

## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] Changes have been rendered and validated locally using mkdocs-material (see edgex-docs README)
- [ ] Docs have been added / updated (for bug fixes / features)

## What is the current behavior?
<!-- Please describe the current behavior and link to a relevant issue. -->
Can be confusing to implement the example.

## Issue Number:
edgexfoundry#546

## What is the new behavior?
Reorganized to improve workflow and reduce confusion around the custom configuration, simulator creation, and edgex services.

## Does this PR introduce a breaking change?
<!-- If this PR contains a breaking change, please describe the impact and migration path for existing applications below. -->

- [ ] Yes
- [x] No

## Specific Instructions
Are there any specific instructions or things that should be known prior to reviewing?
No changes to the source code base, this is improvements to the existing docs.

## Other information

lenny-intel reviewed (changes requested):
Looks good, just one tweak. I think that very last example of the override should change to the  these:
```
      DEVICE_DEVICESDIR: /custom-config/devices
      DEVICE_PROFILESDIR: /custom-config/profiles
 ```
rather than 
```
       MQTTBROKERINFO_HOST: 172.17.0.1
```
Which may be confusing.

lenny-intel commented:
Also looks like you need to rebase and resolve conflicts.

jenmwms commented:
Please re-review. I made these changes:
-Updated very last section on environment variable override (after the table) based on suggested edits.
-Rebased to resolve conflicts.
